DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateGroups/GroupCourseExpandableViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using DigitalLearningSolutions.Data.Models.DelegateGroups;
+
+    public class GroupCourseExpandableViewModel
+    {
+        public GroupCourseExpandableViewModel(GroupCourse groupCourse)
+        {
+            GroupCustomisationId = groupCourse.GroupCustomisationId;
+            Name = groupCourse.CourseName;
+            Supervisor = groupCourse.SupervisorLastName != null

```

Comment:
If the supervisor has only a first name in the db, is there a reason we're not displaying anything?

Reply:
SupervisorLastName is AdminUsers.Surname which is a non-nullable column. If this is null, we don't have a supervisor because the LEFT join to the AdminUsers on SupervisorAdminID (nullable column) has no results

Reply:
Ah cool, that makes sense! 👍